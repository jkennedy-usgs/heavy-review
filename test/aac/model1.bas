# BAS6 package for  MODFLOW-NWT, generated by Flopy.
FREE
OPEN/CLOSE            ./ibound_layer_1.ref               1   (274I10) -1 ibound layer 1                
    -999.0
OPEN/CLOSE              ./strt_layer_1.ref               1 (274E15.6) -1 strt layer 1                  
