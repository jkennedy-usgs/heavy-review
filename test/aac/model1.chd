# CHD package for  MODFLOW-NWT, generated by Flopy.
       189
       189         0 # stress period 1
         open/close ./CHD_0000.dat
       189         0 # stress period 2
         open/close ./CHD_0001.dat
       189         0 # stress period 3
         open/close ./CHD_0002.dat
       189         0 # stress period 4
         open/close ./CHD_0003.dat
       189         0 # stress period 5
         open/close ./CHD_0004.dat
       189         0 # stress period 6
         open/close ./CHD_0005.dat
       189         0 # stress period 7
         open/close ./CHD_0006.dat
       189         0 # stress period 8
         open/close ./CHD_0007.dat
       189         0 # stress period 9
         open/close ./CHD_0008.dat
       189         0 # stress period 10
         open/close ./CHD_0009.dat
       189         0 # stress period 11
         open/close ./CHD_0010.dat
       189         0 # stress period 12
         open/close ./CHD_0011.dat
       189         0 # stress period 13
         open/close ./CHD_0012.dat
       189         0 # stress period 14
         open/close ./CHD_0013.dat
       189         0 # stress period 15
         open/close ./CHD_0014.dat
       189         0 # stress period 16
         open/close ./CHD_0015.dat
       189         0 # stress period 17
         open/close ./CHD_0016.dat
       189         0 # stress period 18
         open/close ./CHD_0017.dat
       189         0 # stress period 19
         open/close ./CHD_0018.dat
       189         0 # stress period 20
         open/close ./CHD_0019.dat
       189         0 # stress period 21
         open/close ./CHD_0020.dat
