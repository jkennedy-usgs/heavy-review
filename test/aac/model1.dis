# DIS package for  MODFLOW-NWT, generated by Flopy.
         1       198       274        21         5         2
  0
OPEN/CLOSE                      ./delr.ref               1 (274E15.6) -1 delr                          
OPEN/CLOSE                      ./delc.ref               1 (198E15.6) -1 delc                          
OPEN/CLOSE                 ./model_top.ref               1 (274E15.6) -1 model_top                     
OPEN/CLOSE              ./botm_layer_1.ref               1 (274E15.6) -1 botm layer 1                  
      1.000000             1  1.000000  SS 
      5.000000             5  1.000000  TR 
     33.000000            33  1.000000  TR 
     21.000000            21  1.000000  TR 
      1.000000             1  1.000000  TR 
      1.000000             1  1.000000  TR 
      1.000000             1  1.000000  TR 
      1.000000             1  1.000000  TR 
      1.000000             1  1.000000  TR 
      1.000000             1  1.000000  TR 
      1.000000             1  1.000000  TR 
      1.000000             1  1.000000  TR 
      1.000000             1  1.000000  TR 
      1.000000             1  1.000000  TR 
      1.000000             1  1.000000  TR 
      1.000000             1  1.000000  TR 
      1.000000             1  1.000000  TR 
      1.000000             1  1.000000  TR 
      1.000000             1  1.000000  TR 
      1.000000             1  1.000000  TR 
      1.000000             1  1.000000  TR 
