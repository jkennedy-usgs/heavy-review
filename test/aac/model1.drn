# DRN package for  MODFLOW-NWT, generated by Flopy.
       135        53
       135         0 # stress period 1
         open/close ./DRN_0000.dat
       135         0 # stress period 2
         open/close ./DRN_0001.dat
       135         0 # stress period 3
         open/close ./DRN_0002.dat
       135         0 # stress period 4
         open/close ./DRN_0003.dat
       135         0 # stress period 5
         open/close ./DRN_0004.dat
       135         0 # stress period 6
         open/close ./DRN_0005.dat
       135         0 # stress period 7
         open/close ./DRN_0006.dat
       135         0 # stress period 8
         open/close ./DRN_0007.dat
       135         0 # stress period 9
         open/close ./DRN_0008.dat
       135         0 # stress period 10
         open/close ./DRN_0009.dat
       135         0 # stress period 11
         open/close ./DRN_0010.dat
       135         0 # stress period 12
         open/close ./DRN_0011.dat
       135         0 # stress period 13
         open/close ./DRN_0012.dat
       135         0 # stress period 14
         open/close ./DRN_0013.dat
       135         0 # stress period 15
         open/close ./DRN_0014.dat
       135         0 # stress period 16
         open/close ./DRN_0015.dat
       135         0 # stress period 17
         open/close ./DRN_0016.dat
       135         0 # stress period 18
         open/close ./DRN_0017.dat
       135         0 # stress period 19
         open/close ./DRN_0018.dat
       135         0 # stress period 20
         open/close ./DRN_0019.dat
       135         0 # stress period 21
         open/close ./DRN_0020.dat
