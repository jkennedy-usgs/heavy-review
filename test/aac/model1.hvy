15  81 -999  # NFHVY IFUN NPMHVY IPMUN HVYNOH
1  56.64047  47897  20608  GAAC-17
1  45.91008  34755  19905  GCOCA
1  45.1254  35381  19411  GGRAY
1  57.60853  42008  23781  GLCWSP3
1  45.12638  35047  19704  GLCWSP4
1  54.91077  37160  20298  GMDWY
1  58.04315  41641  24091  GMW10-2
1  75.90992  44465  27345  GOGIL
1  62.44864  50855  21318  GPK-1
1  46.39929  35913  20105  GT1-S1
1  46.43591  35862  19464  GT1-S4
1  46.0771  35817  19218  GT1-S5
1  47.94432  39152  22515  GT2-S2
1  49.91065  39897  22990  GT3-S2
1  59.22445  50294  21240  GT4-S2
