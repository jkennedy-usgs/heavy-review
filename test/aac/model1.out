 PRLBL               KPER    KSTP         TOTIM        d-uGAL
 GAAC-17                1       1               1.000         0.000         0.000
 GCOCA                  1       1               1.000         0.000         0.000
 GGRAY                  1       1               1.000         0.000         0.000
 GAAC-17                2       1               2.000        37.874        37.874
 GCOCA                  2       1               2.000        38.809        38.809
 GGRAY                  2       1               2.000        29.962        29.962
 GAAC-17                3       1               7.000       103.823       103.823
 GCOCA                  3       1               7.000       111.584       111.584
 GGRAY                  3       1               7.000        99.743        99.743
 GAAC-17                4       1              40.000       275.001       275.001
 GCOCA                  4       1              40.000       287.410       287.410
 GGRAY                  4       1              40.000       276.563       276.563
 GAAC-17                5       1              61.000       336.864       336.864
 GCOCA                  5       1              61.000       334.734       334.734
 GGRAY                  5       1              61.000       325.758       325.758
 GAAC-17                6       1              62.000       339.343       339.343
 GCOCA                  6       1              62.000       336.774       336.774
 GGRAY                  6       1              62.000       327.813       327.813
 GAAC-17                7       1              63.000       341.774       341.774
 GCOCA                  7       1              63.000       338.811       338.811
 GGRAY                  7       1              63.000       329.874       329.874
 GAAC-17                8       1              64.000       344.164       344.164
 GCOCA                  8       1              64.000       340.833       340.833
 GGRAY                  8       1              64.000       331.920       331.920
 GAAC-17                9       1              65.000       346.505       346.505
 GCOCA                  9       1              65.000       342.761       342.761
 GGRAY                  9       1              65.000       333.862       333.862
 GAAC-17               10       1              66.000       348.727       348.727
 GCOCA                 10       1              66.000       344.138       344.138
 GGRAY                 10       1              66.000       335.192       335.192
 GAAC-17               11       1              67.000       350.814       350.814
 GCOCA                 11       1              67.000       333.942       333.942
 GGRAY                 11       1              67.000       328.544       328.544
 GAAC-17               12       1              68.000       316.486       316.486
 GCOCA                 12       1              68.000       327.461       327.461
 GGRAY                 12       1              68.000       323.182       323.182
 GAAC-17               13       1              69.000       300.157       300.157
 GCOCA                 13       1              69.000       296.559       296.559
 GGRAY                 13       1              69.000       298.306       298.306
 GAAC-17               14       1              70.000       288.625       288.625
 GCOCA                 14       1              70.000       280.907       280.907
 GGRAY                 14       1              70.000       282.628       282.628
 GAAC-17               15       1              71.000       279.343       279.343
 GCOCA                 15       1              71.000       269.452       269.452
 GGRAY                 15       1              71.000       271.111       271.111
 GAAC-17               16       1              72.000       271.350       271.350
 GCOCA                 16       1              72.000       259.982       259.982
 GGRAY                 16       1              72.000       261.581       261.581
 GAAC-17               17       1              73.000       264.147       264.147
 GCOCA                 17       1              73.000       251.413       251.413
 GGRAY                 17       1              73.000       252.902       252.902
 GAAC-17               18       1              74.000       257.499       257.499
 GCOCA                 18       1              74.000       243.683       243.683
 GGRAY                 18       1              74.000       245.071       245.071
 GAAC-17               19       1              75.000       251.296       251.296
 GCOCA                 19       1              75.000       236.740       236.740
 GGRAY                 19       1              75.000       238.048       238.048
 GAAC-17               20       1              76.000       245.414       245.414
 GCOCA                 20       1              76.000       226.679       226.679
 GGRAY                 20       1              76.000       227.446       227.446
 GAAC-17               21       1              77.000       239.785       239.785
 GCOCA                 21       1              77.000       214.876       214.876
 GGRAY                 21       1              77.000       214.927       214.927
